site/content/docs/5.1/components/tags.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: docs
+title: Tags
+description: Documentation and examples for tag component.
+group: components
+toc: true
+---
+
+## Examples
+
+A tag is basically a rounded container that can contain some elements. It can be clickable and have a close button.
+
+{{< callout warning >}}
+Each tag version shouldn't be used aside another version of a tag.
+{{< /callout >}}
+
+{{< example class="d-flex gap-2 align-items-center" >}}
+<span class="tag">
+  Label
+</span>
+
+<button class="tag">
+  Label
+</button>
+
+<span class="tag">
+  <svg fill="currentColor" aria-hidden="true" focusable="false" width="1.5rem" height="1.5rem">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  Label
+  <button class="close">
+    <span class="visually-hidden">Close</span>
+  </button>
+</span>
+{{< /example >}}
+
+## With icon
+
+A tag can be build with an icon (img or svg).
+
+{{< example class="d-flex gap-2" >}}
+<div class="tag">
+  <svg fill="currentColor" aria-hidden="true" focusable="false" width="1.5rem" height="1.5rem">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  Label
+</div>
+{{< /example >}}
+
+## Actionable
+
+A tag can be actionable either when builded with a `<button>` tag or with a close button inside.
+
+{{< callout info >}}
+A `<button>` inside a `<button>` is forbidden for accessibility concerns.
+{{< /callout >}}
+
+{{< example class="d-flex gap-2 align-items-center" >}}
+<button class="tag">
+  Label
+</button>
+
+<button class="tag">
+  <svg fill="currentColor" aria-hidden="true" focusable="false" width="1.5rem" height="1.5rem">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  Label
+</button>
+
+<span class="tag">
+  Label
+  <button class="close">
+    <span class="visually-hidden">Close</span>
+  </button>
+</span>
+
+<span class="tag">
+  <svg fill="currentColor" aria-hidden="true" focusable="false" width="1.5rem" height="1.5rem">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  Label
+  <button class="close">
```

Comment:
Not possible to use the Close button component already embedded in Boosted?

Reply:
I don't think so since the behavior isn't the same. Maybe it could if I add a behavior to that close button.